### ISSUE TITLE
Implement nfts api calls (CoinGecko)

### ISSUE THREAD

**Issue by oa-s** [AlphaWallet/alpha-wallet-ios#5925]
CoinGecko now supports endpoints for retrieving floor_price for nfts

https://api.coingecko.com/api/v3/nfts/list

**Comment by hboon** [comment]
Does this work for chains other than mainnet? Otherwise OpenSea covers this right?

**Comment by oa-s** [comment]
Yes, it supports the other chains too, have to check which ones, but go init( CoinGecko platform) for rpc server.

**Comment by oa-s** [comment]
What might be an issue is that it don't have an endpoint to retrieve batch of tickers for nft tokens, might have throttling of requests. Need to check it

**Comment by hboon** [comment]
Ok. Lower priority?

**Comment by JamesSmartCell** [comment]
Nice, if you get it working can you create an Android issue with the endpoints?
